org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/export/ToscaExportUtil.java

```diff
@@ -358,13 +400,15 @@ private void prepareForExport(IRepository repository, ArtifactTemplateId id) thr
      * <p>
      * Thereby, it uses the global variable referencesToPathInCSARMap
      */
-    private void putRefAsReferencedItemInCsar(RepositoryFileReference ref) {
+    private static void putRefAsReferencedItemInCsar(RepositoryFileReference ref, Map<RepositoryFileReference, CsarContentProperties> referencesToPathInCSARMap) {
```

Comment:
Why map as parameter and static?

If "OK" reasons, please add `@parameter` statements.